caravel/migrations/versions/27ae655e4247_make_creator_owners.py

```diff
@@ -11,15 +11,34 @@
 down_revision = 'd8bc074f7aad'
 
 from alembic import op
-from caravel import db, models
+from caravel import db
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import relationship
+from flask_appbuilder import Model
+from sqlalchemy import (
+    Column, Integer, ForeignKey, Table)
+
+Base = declarative_base()
+
+
+class Slice(Base):
+    """Declarative class to do query in upgrade"""
+    __tablename__ = 'slices'
+    id = Column(Integer, primary_key=True)
+
+
+class Dashboard(Base):
+    """Declarative class to do query in upgrade"""
+    __tablename__ = 'dashboards'
+    id = Column(Integer, primary_key=True)
 
 
 def upgrade():
     bind = op.get_bind()
     session = db.Session(bind=bind)
 
-    objects = session.query(models.Slice).all()
-    objects += session.query(models.Dashboard).all()
+    objects = session.query(Slice).all()
+    objects += session.query(Dashboard).all()
```

Comment:
@the-dcruz I added a column datasource_id in Slice and Dashboard. If it uses models.Slice and models.Dashboard the session.query would fail because it couldn't find this column at this migration point. It would be helpful if you can share the stacktrace. Thank you